standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/columnstats/merge/DoubleColumnStatsMerger.java

```diff
@@ -50,5 +50,16 @@ public void merge(ColumnStatisticsObj aggregateColStats, ColumnStatisticsObj new
           + aggregateData.getNumDVs() + " and " + newData.getNumDVs() + " to be " + ndv);
       aggregateData.setNumDVs(ndv);
     }
+
+    aggregateColStats.getStatsData().setDoubleStats(aggregateData);
+  }
+
+  private double getMinValue(DoubleColumnStatsDataInspector aggregateData, DoubleColumnStatsDataInspector newData) {
+    if (!aggregateData.isSetLowValue() && !newData.isSetLowValue()) {
+      return 0;
```

Comment:
If they are both unset, why do we return `0` here instead of leaving it unset? I am not sure whether this will ever happen, but it seems the right behavior.

Reply:
My original logic was that the value for the statistics should be 0 (as we don't have anything for that column yet), but I agree with you, it is nicer to leave it unset in this case. In effect it's the same, but the code is cleaner. Fixed.